provider/cloudflare.go

```diff
@@ -224,34 +225,38 @@ func (p *CloudFlareProvider) submitChanges(changes []*cloudFlareChange) error {
 		}
 		for _, change := range changes {
 			logFields := log.Fields{
-				"record": change.ResourceRecordSet.Name,
-				"type":   change.ResourceRecordSet.Type,
-				"ttl":    change.ResourceRecordSet.TTL,
-				"action": change.Action,
-				"zone":   zoneID,
+				"record":  change.ResourceRecordSet[0].Name,
```

Comment:
Wouldn't this only log the first record? This might a bit confusing for users if you have multiple records.

Reply:
The records are created by the same source file - only the targets are likely to differ, this is why I added the amount of targets as log field. I understand that this way of creating records is confusing, an alternative would possibly be to implement a custom type that can hold multiple targets and not using the type from `cloudflare` package internally for this list.